### ISSUE TITLE
[Bug] Sticky keys still there

### ISSUE THREAD

**Issue by godzinilla** [jincao1/qmk_firmware#7]
### Describe the Bug

The "sticky" keys are still there. The issue comes from the official firmware for this keyboard (as far as I know). The "sticky" keys are still there. The issue comes from the official firmware for this keyboard (as far as I know). The custom one promises that this issue should be gone but it is still there and it happens quite often. I am using RF mode, the receiver is plugged in to usb2.0 port on my motherboard. The PC is placed under the desk. It is about a half of a meter between receiver and keyboard.

### Keyboard Used

Nuphy AIR75v2

### Operating System

Windows 11

### Is AutoHotKey / Karabiner installed

- [ ] AutoHotKey (Windows)
- [ ] Karabiner (macOS)

**Comment by jincao1** [comment]
I make an attempt to try and _reduce_ the sticky keys but from what I know it can still happen. How frequently and how bad it is really depends on the individual case unfortunately.

I don't have the wireless firmware source and whatnot so the current state is this is the best I can do, with hopes the vendor addresses it eventually (not very hopeful there). Sorry it doesn't work out for you. Maybe try another firmware on GitHub, you might get better experience.

**Comment by godzinilla** [comment]
It happens rather rarely now but it's still there. If there is nothing more that you can do with it then it's ok. It's still a way better than the original firmware. And it really suck imho. I have owned some Keychron RF keyboards and this didn't happened at all.

**Comment by jincao1** [comment]
Yeah it's a shame... you could try one of the several new RF firmware that Nuphy published. Some have stated it helped while it made no difference for others.

**Comment by godzinilla** [comment]
You mean this firmware?

https://github.com/jincao1/qmk_firmware/tree/air75v2-sleep/keyboards/nuphy/air75_v2/ansi/rf_firmware

Could I flash this using the QMK Toolbox, or only the Android app referred in the manual?
Could I verify what RF firmware I am currently running?
Should I backup my current layout before flashing the RF fw?
Should I flash your fw after flashing the RF?

**Comment by jincao1** [comment]
Mobile is the easiest way. You can flash older versions but can't tell what is currently on the board and to have it backed up. So if the issue is minimal and rare, I'd just leave it.

**Comment by godzinilla** [comment]
> Mobile is the easiest way. You can flash older versions but can't tell what is currently on the board and to have it backed up. So if the issue is minimal and rare, I'd just leave it.

I've flashed the newest one and it still working ;) No issues for now but more testing is needed. Thank you for your help, again :)

**Comment by benjamincburns** [comment]
@godzinilla how's it going so far? also which version of the RF firmware did you try?

**Comment by godzinilla** [comment]
> @godzinilla how's it going so far? also which version of the RF firmware did you try?

Hello, sorry for delayed reply. To be honest not bad but not great either. I've flashed the 1.1.2 RF firmware. The sticky keys are still present, let's say once-twice a day, maybe more (10+h PC usage on RF mode). I've also encountered problems with not picking the key press when leaving the sleep mode and it happens quite often. And today my keyboard hangs on with the "pressed" Enter key (after powering it on and about an hour from the powering on).
